install/linux/usr/share/odemis/examples/sparc2-time-correlator.odm.yaml

```diff
@@ -418,8 +418,8 @@ SPARC2-VisNIR-TR: {
         zero_cross: [10.e-3, 10.e-3], # V (< 20 mV)
         shutter_axes: {
         # internal child role of the actuator -> axis name, position when shutter is closed (ie, protected), position when opened (receiving light)
-            "shutter0": ["x", 0, 1],
-            "shutter1": ["x", 0, 1],
+            "shutter0": ["x", 1, 0],
```

Comment:
Are you sure about this? I feel like it should be the other way around.

Reply:
It's all a matter of perspective. I'm trying to enforce the convention that an "active" shutter is shut/closed. So 1 = closed = no light, and 0 = open = light. 

I think that in an ideal world, the English word for shutter would be something like "apperture-opener" and so the active state (1) would let the light go. For now, I've essentially inverted all the values, to handle this convention that an active shutter blocks the signal.